Gemfile

```diff
@@ -12,6 +12,7 @@ group :test do
   gem "omniauth-openid", "~> 1.0.1"
   gem "webrat", "0.7.3", require: false
   gem "mocha", "~> 1.1", require: false
+  gem 'test_after_commit', require: false
```

Comment:
There are two ways of avoiding using this gem I can think of now:

1. Manually patch AR to support transnational fixtures and callbacks, which is pretty much the gem does, but we can remove the dependency.
2. Disable transactional fixtures, this is the cleanest way I think, but need a lot of work because the records are not deleted after each spec and they will affect each other.

@ulissesalmeida What do you think, which one do you prefer ?